src/components/ExecuteButton.js

```diff
@@ -90,6 +95,7 @@ const ExecuteButton = ({
           issueCount={issueCount}
           runRemediation={runRemediation}
```

Comment:
We can remove `runRemediation` from here since ExecuteModal has no such prop anymore

Reply:
And also from propTypes on line 105